test/common/index.js

```diff
@@ -746,3 +749,47 @@ exports.getTTYfd = function getTTYfd() {
   }
   return tty_fd;
 };
+
+// exports.tag*: tag functions for tagged template literals
+
+// Removes all literal line breaks and leading spaces
+exports.tagGlue = function tagGlue(strings, ...expressions) {
+  const breaks = /\n */g;
+
+  return expressions.reduce(
+    (acc, expr, i) => `${acc}${expr}${strings[i + 1].replace(breaks, '')}`,
+    strings[0].replace(breaks, '')
+  );
+};
+
+// Replaces all literal line breaks and leading spaces with a space, then trims
+exports.tagUnwrap = function tagUnwrap(strings, ...expressions) {
+  const breaks = /(\n *)+/g;
+
+  return expressions.reduce(
+    (acc, expr, i) => `${acc}${expr}${strings[i + 1].replace(breaks, ' ')}`,
+    strings[0].replace(breaks, ' ')
+  ).replace(/^ | $/g, '');
```

Comment:
On this stage, the replacing concerns the whole string, including interpolated variables, so I prefer to be extra careful here. i.e. to remove only two auxiliary spaces (which were framing line breaks before the reducing).